test/unit/math/prim/mat/prob/VectorRNGTestRig.hpp

```diff
@@ -42,9 +42,9 @@
  * input vectors. These should cause invalid_argument errors.
  *
  * The output of generate_samples must be of
- *   stan::length(out) == stan::length(p1) if only p1 is used
- *   stan::length(out) == stan::max_size(p1, p2) if p1 and p2 are used
- *   stan::length(out) == stan::max_size(p1, p2, p3) if all parameters are used
+ *   size(out) == stan::math::size(p1) if only p1 is used
+ *   size(out) == stan::math::max_size(p1, p2) if p1 and p2 are used
+ *   size(out) == stan::math::max_size(p1, p2, p3) if all parameters are used
```

Comment:
This will need to be changed if you change the namespace where `max_size` lives in. Same for similar changes in tests.